code/modules/reagents/newchem/other.dm

```diff
@@ -252,12 +254,21 @@ datum/reagent/super_hairgrownium
 	result_amount = 3
 	mix_message = "The liquid becomes amazingly furry and smells peculiar."
 
-datum/reagent/super_hairgrownium/on_mob_life(var/mob/living/M as mob)
-	if(!M) M = holder.my_atom
+datum/reagent/super_hairgrownium/reaction_mob(var/mob/living/M, var/volume)
```

Comment:
Means that super hairgrownium won't wreck Vox and Plasmamen with respawning fake moustaches that keep spawning on their face until the reagent is out of their system. It now only spawns one fake moustache and triggers a hair change just once as well.